frontend/integration-tests/views/secrets.view.ts

```diff
@@ -125,3 +125,8 @@ export const isValueInJSONPath = (propertyPath: string, value: string, name: str
   const result = _.get(resource, propertyPath, undefined);
   return result === value;
 };
+
+export const getPathInJSON = (path: string, name: string, namespace: string, kind: string) => {
```

Comment:
I'd change this to simply return the object instead of taking in a path. Then the caller can use the object in any way.

Reply:
Sounds much better, I will update